libs/feature/dataviz/src/lib/figure/figure-container/figure-container.component.ts

```diff
@@ -4,7 +4,7 @@ import {
   Input,
   OnChanges,
 } from '@angular/core'
-import { TableItemModel } from '@geonetwork-ui/ui/layout'
+import { TableItemModel } from '@geonetwork-ui/ui/dataviz'
```

Comment:
You are not leveraging the capacity of standalone component are you ?

If you do that, you keep passing through modules to import your deps, so you bypass standalone import.

Reply:
I might be missing something, but I thought removing the standalone component from the module exports was sufficient. This allows to import the standalone component without the entire module. The export in the `index.ts` still seems necessary to me to import the component somewhere, isn't it?
I chose to make `TableComponent` and `ChartComponent` standalone as they have various dependencies/imports and keep the `FiguerComponent` a common one as it only imports the generic `MatIconModule` which will likely be used by other dataviz components.

Reply:
Do you think I should also get rid of the standalone components in the `index.ts` exports and import them via relative paths?

Reply:
Importing via relative path is forbidden by NX rules so... I guess there's no choice

Reply:
I saw somewhere that last version of Nx was proposing a bootstrap based on standalone components.
We should try to have a look about what they propose soon.

Note that you import `TableComponent` module from `FeatureDataviz` module while you might should  import it from `FigureContainerComponent` instead shouldn't you ?
See https://angular.io/guide/standalone-components#the-standalone-flag-and-component-imports

Anyway, it's quite new so we should be careful about using this. I think it's the good direction but we have to do it the proper way.
You can merge the PR but we should figure out the best practices out of it.

Reply:
Indeed, I still have a doubt if the bootstrapping needs further fine-tuning to minimize bundle size. I'll try to do a test with the `ChartComponent`.

Regarding the import of the `TableComponent` into another component (smart ones in our cases), this enforces the importing component to become a standalone component as well, as only standalone components can import standalone components. Not sure if this is the way to go. It would have a larger impact on the architecture in any case.

Reply:
We'll talk about that later, I think that yes, it should be a standalone component too. We need to define a clear strategy for further additions.
Nvm, all of this won't probably change the size of `gn-wc.js` :disappointed: